lib/blacklight/document/active_model_shim.rb

```diff
@@ -14,6 +14,11 @@ def primary_key
       def base_class
         self
       end
+
+      def find id
+        response = Blacklight.default_index.find id
+        self.new(response.documents.first, response)
```

Comment:
Do we need the `self.new` business? `response.documents.first` should be equivalent (unless we're worried about casting between document types, i suppose?)